bin/trivy_validator.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+import json
+import os
+from pathlib import Path
+
+git_root = [x for x in Path(__file__).resolve().parents if (x / ".git").is_dir()][-1]
```

Comment:
The best version I can find of this assignment is:

```suggestion
git_root = next(iter([x for x in Path(__file__).resolve().parents if (x / ".git").is_dir()]), None)
```